plugins/@grouparoo/mailchimp/__tests__/test.ts

```diff
@@ -0,0 +1,37 @@
+import { helper } from "@grouparoo/spec-helper";
+
+import { SimpleAppOptions } from "@grouparoo/core";
+import { test as testConnection } from "../src/lib/test";
+import { loadAppOptions, updater } from "./utils/nockHelper";
+
+const { newNock } = helper.useNock(__filename, updater);
+
+describe("mailchimp/test", () => {
+  helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
+
+  test("Can connect with an API Key", async () => {
+    const appOptions: SimpleAppOptions = loadAppOptions(newNock, false);
+
+    const { success } = await testConnection({
+      app: null,
+      appId: null,
+      connection: null,
+      appOptions,
+    });
+
+    expect(success).toBe(true);
+  });
+
+  test("Can connect with an OAuth Token", async () => {
+    const appOptions: SimpleAppOptions = loadAppOptions(newNock, true);
+
+    const { success } = await testConnection({
```

Comment:
Because we're using a Mailchimp client library for everything and do not have custom methods, it seems like testing if we can connect is enough to know if oauth is doing its job

Reply:
+1

Reply:
I was about to say that `__tests__/test` is a silly name for a file, but it makes sense!

Reply:
Under the hood, this also checks that the DC can be looked up 👍